src/tests/server/database/user.test.js

```diff
@@ -1,92 +1,133 @@
+  const user = createUser();
+
+  Models.Admin.create({ user_id: user.id });
+
+  return user;
+};
+
 describe(`user tests`, async () => {
   before(async () => await initTestDatabase());
+  afterEach(() => clearTestData());
   after(() => concludeTesting());
 
   /*
   export function processUserLogin(userObject) {
   */
 
   test(`deleteUser`, () => {
-    const user = Models.User.create({ name: `disposable` });
+    const user = createUser();
     User.deleteUser(user);
   });
 
   test(`enable/disable`, () => {
-    let user = User.getUser(`test-user`);
+    let user = createUser();
+    const username = user.name;
 
     User.enableUser(user);
-    user = User.getUser(`test-user`);
+    user = User.getUser(username);
     assert.notEqual(user.enabled_at, null);
 
     User.disableUser(user);
-    user = User.getUser(`test-user`);
+    user = User.getUser(username);
     assert.equal(user.enabled_at, null);
   });
 
   test(`getAllUsers`, () => {
+    createUser();
+    createUser();
     const users = User.getAllUsers();
     assert.equal(users.length, 2);
   });
 
   test(`userIsAdmin`, () => {
-    const admin = User.getUser(`test-admin`);
-    assert.equal(User.userIsAdmin(admin), true);
+    const admin = createAdminUser();
+    const user = createUser();
 
-    const user = User.getUser(`test-user`);
+    assert.equal(User.userIsAdmin(admin), true);
     assert.equal(User.userIsAdmin(user), false);
   });
 
   test(`getUserSettings`, () => {
-    const admin = User.getUser(`test-admin`);
+    const admin = createAdminUser();
     let settings = User.getUserSettings(admin);
     assert.deepEqual(settings, {
-      name: `test admin`,
+      name: admin.name,
       admin: true,
       enabled: true,
       suspended: false,
     });
 
-    const user = User.getUser(`test-user`);
-    const s = User.suspendUser(user, `why not`);
+    const user = createUser();
+    User.suspendUser(user, `why not`);
     User.disableUser(user);
     settings = User.getUserSettings(user);
     assert.deepEqual(settings, {
-      name: `test user`,
+      name: user.name,
       admin: false,
       enabled: false,
       suspended: true,
     });
   });
 
   test(`getUserSuspensions`, () => {
-    const user = Models.User.create({ name: `sus user` });
+    const user = createUser();
     const s = User.suspendUser(user, `why not`);
     User.unsuspendUser(s.id);
-    const t = User.suspendUser(user, `why not, again`);
+    User.suspendUser(user, `why not, again`);
     let list = User.getUserSuspensions(user);
     assert.equal(list.length, 1);
     list = User.getUserSuspensions(user, true);
     assert.equal(list.length, 2);
   });
 
   test(`hasAccessToUserRecords`, () => {
-    const admin = User.getUser(`test-admin`);
-    const user = User.getUser(`test-user`);
-    const rando = Models.User.create({ name: `rando calrisian` });
+    const admin = createAdminUser();
+    const user = createUser();
+    const rando = createUser();
```

Comment:
Lots of users getting created: let's add some names

```suggestion
    const admin = createAdminUser(`test-admin`);
    const user = createUser(`test-user`);
    const rando = createUser(`random-user`);
```

Reply:
@Pomax  ahhh okay, I misunderstood! I thought the goal was to remove explicit names to prevent collisions, but adding custom names makes sense!

thank you for all the feedback, super appreciate it! I figure this will be a nice trial and then I can tackle other tests in a similar manner